frontend/src/pages/_app.tsx

```diff
@@ -2,6 +2,8 @@ import type { AppProps } from "next/app";
 import { ChakraProvider } from "@chakra-ui/react";
 import { AuthProvider } from "@/utils/auth";
 
+import "@/components/CodeRoom/CodeEditor/CodeEditor.css";
```

Comment:
I haven't had the chance to pull the PR and run the code to test. It's possible that I've missed something, but I'm wondering if the CSS classes in `CodeEditor.css` are being used?

Reply:
Here's what it looks like following this addition:

<img width="165" alt="Screenshot 2023-11-09 at 10 15 19 PM" src="https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g36/assets/813865/f32de229-682e-47f0-a450-f2e3384cb7f7">

I got the CSS from the yjs-monaco demo: https://github.com/yjs/y-monaco/blob/81e6d81c4ac2ba0e80ce3d176cc5bf0c76cec4d9/demo/index.html#L12-L30.

Reply:
I see... thanks for explaining!